src/util-memcpy.h

```diff
@@ -18,7 +18,7 @@
 /**
  * \file
  *
- * \author Ken Steele <suricata@tilera.com>
+ * \author Victor Julien <vjulien@oisf.net>
```

Comment:
Was the removal of the other author intended?

Reply:
I'll remove both. I don't think it actually adds much over git